.github/workflows/commitlint.yml

```diff
@@ -31,6 +41,13 @@ jobs:
         env:
           PR_TITLE: "${{ github.event.pull_request.title }}"
 
+      - name: SwayLint
+        run: |
+          pnpm --filter starboard/contracts build:pre
```

Comment:
why do we need pnpm here? Can't we just run forc fmt?

Reply:
Some generated code for tests is needed because fmt complains on missing files without it.